src/Controller/BlogController.php

```diff
@@ -71,6 +68,34 @@ public function formArticle(Request $request, EntityManagerInterface $manager, A
         ]);
     }
 
+    #[Route('/updateArticle/{id}', name: 'updateArticle')]
+    public function updateArticle($id, Request $request, EntityManagerInterface $manager)
+    {
+        $form = $this->createForm(ArticleCreateType::class);
+        $form->handleRequest($request);
+
+        $entityManager = $this->getDoctrine()->getManager();
+        $article = $entityManager->getRepository(Article::class)->find($id);
+
+        if (!$article) {
+            throw $this->createNotFoundException('No product found for id '.$id);
```

Comment:
Maybe "article" instead of "product" should be better ? ^,
```suggestion
            throw $this->createNotFoundException('No article found for id '.$id);
```